samples/DevTestLabs/SimpleUI/ClientApp/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "simple_dtl_ui",
+  "name": "simple_dtl_ui",
```

Comment:
If this is the display name - can make it nicer...  If the file isn't needed - can remove.